src/sdk-reference/js/6/getting-started/react/snippets/search.js

```diff
@@ -0,0 +1,5 @@
+const res = await kuzzle.document.search(
```

Comment:
Nitpicking: Usually we named the `document:search` return `results` because it's a `SearchResults` instance